src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/listeners/CollectFailedTestLogsListener.groovy

```diff
@@ -28,7 +30,7 @@ class CollectFailedTestLogsListener extends AbstractSpringListener{
 
     @Override
     void error(ErrorInfo error) {
-        if (!isFailedInPreTest(error)) {
+        if (!isFailedInPreTest(error) && profile == "virtual") {
```

Comment:
why only for vortual?

Reply:
There is no need to collect logs for profile=hardware as we can obtain them after tests execution. As for the "virtual", all logs will be unavailable due to the JJ cleanup.